src/extras/core/Interpolations.d.ts

```diff
@@ -0,0 +1,5 @@
+export namespace Interpolations {
+	export function CatmullRom(): number;
```

Comment:
I think you should also add the function parameters.

Reply:
@Mugen87 I think it's OK now, Please check it again.